ludwig/schema/model_config.py

```diff
@@ -198,6 +198,9 @@ def __init__(self, config_dict: ModelConfigDict):
         self._set_hyperopt_defaults()
 
         # ===== Validate Config =====
+        if self.model_type == MODEL_GBM:
```

Comment:
@connor-mccorm In order to get this to work with existing tests, I have to prevent `self.combiner` from getting set/serialized in the config object when the model type is GBM. Otherwise in some tests there are (presumably empty/ignored) `combiner` configs that are not allowed by this PR. Let me know if there is a better way to do this.

Reply:
This sounds good to me!